SmartDeviceLink-iOS/SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -185,10 +185,9 @@ - (void)createIAPControlSessionWithAccessory:(EAAccessory *)accessory {
 
     if (self.controlSession) {
         self.controlSession.delegate = self;
-
-        if (self.protocolIndexTimer == nil) {
-            self.protocolIndexTimer = [[SDLTimer alloc] initWithDuration:protocolIndexTimeoutSeconds];
-        }
+        
+        // This will restart the timer if it already exists
+        self.protocolIndexTimer = [[SDLTimer alloc] initWithDuration:protocolIndexTimeoutSeconds];
 
```

Comment:
Do you have any more thoughts? If you think it would be better to be explicit about the `cancel` call, I can work that in. It shouldn't be too bad.